Data-Science/README.md

```diff
@@ -0,0 +1,176 @@
+- [Data Visualization python -Youtube(Intellipaat)](https://www.youtube.com/live/_YWwU-gJI5U?feature=share)
+- [Data Visualization -Youtube(datacamp)](https://app.datacamp.com/learn/courses/introduction-to-data-visualization-with-matplotlib?fbclid=IwAR1OrJSdZ2LVD_c1o3d-_1I7Nhq8OZ3pzTu4010E_XWEmMc0KYsTosz8CIU)
+- [Data Visualization -Kaggle(kaggle)](https://www.kaggle.com/learn/data-visualization)
+
+### EDA
+
+- [EDA and Feature Engineering -Youtube(Krish Naik)](https://youtu.be/fHFOANOHwh8)
+- [EDA in Python -Youtube(Simplilearn)](https://youtu.be/MoM6mighOJM)
+- [EDA -datacamp(datacamp.com)](https://learn.datacamp.com/courses/exploratory-data-analysis-in-python)
+- [EDA for ML -coursera(coursera.com)](https://www.coursera.org/learn/ibm-exploratory-data-analysis-for-machine-learning)
+
+### Machine Learning
+
+- [ML for beginner -Youtube(freeCodeCamp.org)](https://youtu.be/NWONeJKn6kc)
+- [ML full course playlist -Youtube(Andrew Ng)](https://youtube.com/playlist?list=PLoROMvodv4rMiGQp3WXShtMGgzqpfVfbU)
+- [ML full playslist -Youtube(KunalKushwaha)](https://youtube.com/playlist?list=PLyzHIYrZBplo3K0dNUqppd2ynnoZPD6N1)
+- [ML full course video -Youtube(Simplilearn)](https://www.youtube.com/live/lTzHlU3OrXs?feature=share)
+- [ML full playlist -Youtube(Kimia Lab)](https://youtube.com/playlist?list=PL4upCU5bnihwCX93Gv6AQnKmVMwx4AZoT)
+- [ML documentation GFG(GeeksForGeeks)](https://www.geeksforgeeks.org/machine-learning/)
+- [ML intro to machine learning course -Udacity(udacity)](https://www.udacity.com/course/intro-to-machine-learning--ud120)
+- [ML with pythonn -Coursera(IBM)](https://www.coursera.org/learn/machine-learning-with-python)
+
+### Power BI
+
+- [Power BI training -Microsoft](https://powerbi.microsoft.com/en-us/learning/)
+- [Power BI full course tutorial -Youtube(Learnit Training)](https://youtu.be/e6QD8lP-m6E)
+- [Power BI complete course -Coursera(Keith Gibson)](https://www.coursera.org/learn/data-driven-decisions-with-power-bi)
+- [Power BI documented tutorial -GFG(geeksforgeeks)](https://www.geeksforgeeks.org/power-bi-tools-and-functionalities/)
+
+### Tableau
+
+- [Tableau data visualisation tutorial -datacamp(Parul Pandey)](https://www.datacamp.com/tutorial/data-visualisation-tableau)
+- [Tableau training -Salesforce](https://www.tableau.com/learn/training)
+- [Tableau Introduction -datacamp](https://learn.datacamp.com/courses/introduction-to-tableau)
+- [Tableau full course -Youtube(simplilearn)](https://youtu.be/HM81vShSlWY)
+- [Tableau documented tutorial -GFG(geeksforgeeks)](<(https://www.geeksforgeeks.org/what-is-tableau-and-its-importance-in-data-visualization/)>)
+
+### Web Scraping/APIs
+
+- [Web Scraping in Python -datacamp](https://learn.datacamp.com/courses/web-scraping-with-python)
+- [Build Web Scraper with python -RealPython(Martin Breuss)](https://realpython.com/beautiful-soup-web-scraper-python/)
+- [Web Scraping with python -Youtube(freeCodeCamp.org)](https://youtu.be/XVv6mJpFOb0)
+
+### Deep Learning
```

Comment:
Deep Learning, PyTorce, etc. are ML concepts and doesn't require for Data Science. While I understand that scikit-learn goes hand in hand with these topics, scikit learn from the ML Path is enough for introduction.